### DISCUSSION TITLE
M5Stack library and MCP2515

### DISCUSSION THREAD

**Discussion by bbjodel** [Bodmer/TFT_eSPI#2686] [General]
Hello everyone,
I have an ESP32 WROOM 32 with a TFT ILI9341 SPI and a CAN MCP2515 SPI, but I can't operate the CAN if the command m5.begin (); is active in the code, if it is //m5.begin (); The CAN works.
My question is how to make the TFT Ili9341 spi with the CAN MCP2515 cohabit?

Ili9341:
Miso (not connected) otherwise GPIO19
Mosi ===> GPIO23
CS =====> GPIO15
DC =====> GPIO2
Reset ===> GPIO4
SCLK ===> GPIO18

MCP2515:
Miso ===> GPIO19
Mosi ===> GPIO23
CS =====> GPIO15
DC =====> GPIO5
Reset ====> GPIO4
SCLK ===> GPIO18

I tried everything but without success it looks like the M5Stack library is incompatible and blocks the CAN.
Can you help me ?

Regards,
Alfred from France, sorry for my english....

**Comment by Bodmer** [comment]
This question should be asked in the M5Stack library respository!

**Reply by bbjodel** [reply]
OK Thank you
